src/app/store/reference-data/reducers/reference-data.reducer.ts

```diff
@@ -19,7 +26,12 @@ interface ReferenceDataEntityState extends EntityState<ReferenceDataModelBase> {
   loading: boolean;
 }
 
-export type ReferenceDataState = Record<ReferenceDataResourceType, ReferenceDataEntityState>;
+export interface ReferenceDataEntityStateTyres extends EntityState<ReferenceDataModelBase> {
+  loading: boolean;
+  searchReturn: any[] | null;
+}
```

Comment:
```suggestion
export interface ReferenceDataEntityStateTyres extends ReferenceDataEntityState {
  searchReturn: any[] | null;
}
```

The loading flag is already in the ReferenceDataEntityState